honeybadger/contrib/django.py

```diff
@@ -66,12 +66,19 @@ def generate_payload(self, default_payload, config, context):
         :param config: honeybadger configuration.
         :return: a dict with the generated payload.
         """
-        request = current_request()
+        import django
+        if django.__version__.startswith('1.11'):
```

Comment:
What does this mean?
Shouldn't we check for `django version < 2` here ?

Reply:
hmm, maybe
I just copied it from [here](https://github.com/honeybadger-io/honeybadger-python/blob/master/honeybadger/tests/contrib/django_test_app/urls.py#L5)

Reply:
you're right, I got confused with [that](https://github.com/honeybadger-io/honeybadger-python/blob/master/honeybadger/tests/contrib/django_test_app/urls.py#L5).
Updated.